src/GNDStk/Node/src/call-child.hpp

```diff
@@ -24,15 +24,39 @@ decltype(auto) operator()(
    std::string &&label,
    bool &found = detail::default_bool
 ) GNDSTK_CONST {
+   // as above, don't need or want (kwd.name == "") conditional
    try {
-      // As above, don't need or want:
-      //    if (kwd.name == "")
-      //       detail::apply_converter<TYPE>{}(kwd,*this);
-      return child(kwd.one(), detail::label_is(label), found);
+      // total filter
+      auto filter = [kwd,label](const Node &n)
+         { return kwd.filter(n) && detail::label_is(label)(n); };
+      // --kwd: child_t<...,allow::one,...>
+      return child(--kwd+filter, found);
```

Comment:
Isn't it nice when things come together :-)

Reply:
Definitely. Build a solid foundation, and adding new features becomes easier and easier.